src/main/java/com/eternalcode/parcellockers/util/DurationUtil.java

```diff
@@ -18,8 +18,8 @@ public static String format(Duration duration, boolean removeMillis) {
         }
 
         return duration.toString()
```

Comment:
When you convert to eternalcode-commons, you can use the time formatting that is added there.

Reply:
Will do in another, related PR.